Sources/daylight-ios/daylight_ios.swift

```diff
@@ -0,0 +1,3 @@
+struct daylight_ios {
+    var text = "Hello, World!"
+}
```

Comment:
What's going on in this file?

Reply:
No idea how this got here, will remove it